packages/ui/src/components/date-time-input/picker/index.tsx

```diff
@@ -93,24 +93,20 @@ export const DateTimePicker = ({
 
     // avoid inconsistent min and max values
     useEffect(() => {
-        setMax((prevMax) => {
-            const newMax = getUpdatedMinMaxValue(prevMax, maxDate);
+        const updatedMin = getUpdatedMinMaxValue(min, minDate);
+        const updatedMax = getUpdatedMinMaxValue(max, maxDate);
 
-            setMin((prevMin) => {
-                let newMin = getUpdatedMinMaxValue(prevMin, minDate);
-                if (dayjs(newMin).isAfter(dayjs(newMax), "seconds")) {
-                    newMin = newMax;
-                    console.warn("inconsistent min and max values", {
-                        min: newMin?.toISOString(),
-                        max: newMax?.toISOString(),
-                    });
-                }
-                return newMin;
-            });
+        setMin(updatedMin);
```

Comment:
Wondering if these `setState`s, can trigger an infinite loop since they update `min/max` values, that are also in the dependency array of the `useEffect`.

Reply:
Yes but the first time they're triggered when a min or max date changes, and those are set in the state, triggering the second run. In the second run the state and passed min and max dates are the same so nothing happens and no useEffect is triggered anymore